frog/imports/client/SingleActivity/Finish.js

```diff
@@ -37,11 +37,13 @@ function Finish(
       </IconButton>
       <Typography variant="h3" component="h2" className={classes.padded_text}>
         You're all set, please share this link with the participants:{' '}
```

Comment:
Remove `{' '}` and simply add a space

Reply:
https://stackoverflow.com/questions/38547558/why-do-people-put-in-their-react-jsx